odata-csdl/14 Vocabulary and Annotation.md

```diff
@@ -1699,7 +1699,8 @@ types](#BuiltInTypesfordefiningVocabularyTerms)
 `Edm.NavigationPropertyPath`, `Edm.AnyPropertyPath`, or `Edm.ModelElementPath`.
 Its argument is a [model path](#PathExpressions) with the following
 restriction:
-- A non-null path MUST resolve to a [navigation property](#NavigationProperty).
+- A non-null path MUST end with a [navigation property](#NavigationProperty)
+or a term cast to a term whose type is an entity type or a collection of entity types.
```

Comment:
Are type casts not allowed here?

Reply:
No, don't see a use case. We want to reference a model element, either a "real" navigation property or an "annotated" one.

Type-casts in model paths only make sense before a property segment to reference a property of a derived type.